tests/link/numba/test_slinalg.py

```diff
@@ -720,3 +722,40 @@ def test_lu_solve(b_func, b_shape: tuple[int, ...], trans: bool, overwrite_b: bo
 
     # Can never destroy non-contiguous inputs
     np.testing.assert_allclose(b_val_not_contig, b_val)
+
+
+@pytest.mark.parametrize("stride", [1, 2, -1], ids=lambda x: f"stride={x}")
```

Comment:
I don't like to parametrize like this for different values, it's wasteful to compile a function for every value we want to test if the types don't change. 

We can extend the helper to allow lists of values or sidestep it or use the numba compiled function that's returned as the first input  to test different strides afterwards.

If you really want parametrize use a fixture to get the compiled function

Reply:
I like the new way a lot less (I don't like that the test fails on the first failure and we don't get info about the remaining cases), but the fixture was giving strange cache warnings I didn't want to deal with.

The compile time on each test is basically zero, I'm not sure I'm on board with this criticism